README.md

```diff
@@ -190,6 +197,10 @@ locally via `--local-json`). The format is as below:
 ]
 ```
 
+The `--local-json` can also be involved via stdin like:
```

Comment:
I think "involved" isn't the fittest word here:

```
The `--local-json` option can also read from `stdin`:
```

Reply:
I adapted the suggestion.